lib/koans/10_map_sets.ex

```diff
@@ -0,0 +1,62 @@
+defmodule MapSets do
+  use Koans
+
+  @intro "My name is Set, MapSet."
+
+  @set MapSet.new([1, 2, 3, 4, 5])
+
+  koan "I am merely another collection but, you can perform some operations on me" do
+    new_set = MapSet.new(@set, fn x -> 3 * x end)
```

Comment:
I think we might want to move both Enum and Functions up the list at some point, so I wouldn't be too fuzzed about it.